templates/namerctl-job.yaml

```diff
@@ -10,20 +10,20 @@ spec:
       name: namerctl
     spec:
       volumes:
-        - name: namerctl-script
-          configMap:
-            name: namerctl-script
-            defaultMode: 0755
+      - name: namerctl-script
+        configMap:
+          name: namerctl-script
+          defaultMode: 0755
       containers:
-        - name: namerctl
-          image: linkerd/namerctl:0.8.6
-          env:
-            - name: NAMERCTL_BASE_URL
-              value: http://namerd.infra.svc.cluster.local:4180
-          command:
-            - "/namerctl/createNs.sh"
-          volumeMounts:
-            - name: "namerctl-script"
-              mountPath: "/namerctl"
-              readOnly: true
+      - name: namerctl
+        image: linkerd/namerctl:0.8.6
+        env:
+        - name: NAMERCTL_BASE_URL
+          value: http://namerd.infra.svc.cluster.local:4180
```

Comment:
This URL needs to take into account the actual namespace we're deployed into.

Reply:
Currently the namespace is hard coded to `infra` in all the namerd charts.. so I did not change this URL.
But I can make the namespace configurable. That'd be better?

Reply:
That's fine. Leave this. We can make the namespace configurable in another PR which does just that.